docs/modules/servers/pages/local/self-hosting.notes

```diff
@@ -0,0 +1,90 @@
+The name of the project is "Jame Self Hosting Sandbox", and I see the term "self hosting" in a few places... So it sounds like it is important, but what is that intended to mean?
+
+I would like to understand better the intention behind the term.
+
+
+
+I would say "self hosting" means individuals having bought a domain name and want to install their own mail server for hosting their personal emails. Volumetry is small. They want control on their data.
+
+One generally tricky point about this kind of set up is getting third parties (like eg GMail) trusting your setup. It would be important to document a checklist regarding Mail Exchange trust like:
+
+ I'm not an open spam relay
+ MX record registered for my domain
+ Reverse DNS setted up for MX servers
+ SPF records setted up
+ DKIM signature setted up
+(which is slightly external to James as DNS setup is required)
```

Comment:
You are perfectly right, we need in my opinion a "concept page" on this topic, and a checklist (on the form of "how to") is a must have for most servers IMO.

Many user questions burns down to that very point, hence documenting a bit further than just "James" on this topic would help.

We should point to external resources to help automate such checks.

https://mxtoolbox.com/diagnostic.aspx would help doing such checks.

Reply:
Thank you. I added your comment to the notes.